src/connection/index.js

```diff
@@ -54,14 +54,14 @@ const connectionDefinitions = (
       # A cursor for use in pagination.
       cursor: String!
       # The item at the end of the edge
-      node: ${connectionNode}
+      node: ${connectionNode}!
```

Comment:
I can't really see a way to make the list *not* have a node inside it.

Reply:
If you want to make the node non-null then you can specify a `nodeType` in the configuration that is nonnull e.g. `User!`.